src/FormattedText.tsx

```diff
@@ -53,7 +53,9 @@ const renderMarkdown = (
     text: string,
     onImageLoad: () => void
 ) => {
-    const src = text.replace(/<br\s*\/?>/ig, '\r\n\r\n');
+    const src = text
+                 .replace(/<br\s*\/?>/ig, '\r\n\r\n')
+                 .replace(/\[(.*)\]\((.*)\)/ig, (match, text, url) => `[${text}](${markdownIt.normalizeLink(url)})`);
```

Comment:
can you add a comment about what this regex is searching for? For the regex-challenged folks like myself :-)